oozie-to-airflow/o2a.py

```diff
@@ -16,14 +16,17 @@
 import argparse
 import os
 import sys
+import logging
 
 from converter.oozie_converter import OozieConverter
 from converter.mappers import ACTION_MAP, CONTROL_MAP
 
 INDENT = 4
+CONFIGURATION_PROPERTIES = "configuration.properties"
```

Comment:
It seems that this value is used in other places.
Could you update other place?
Reference:
https://github.com/GoogleCloudPlatform/cloud-composer/blob/147ca1cb6f38101d6d2968a645cf1cb38ca193b6/oozie-to-airflow/converter/oozie_converter.py#L73

Reply:
Done